packages/utils/src/time-utils/timeUtils.ts

```diff
@@ -0,0 +1,59 @@
+import type { MaybeNumber, RundownCached } from 'ontime-types';
+import { isOntimeEvent } from 'ontime-types';
+
+import { dayInMs } from '../date-utils/conversionUtils';
+
+export function calculateExpectedStart(
```

Comment:
this is a lot of work!
i wonder if our current normalised rundown should be doing this instead?